bower.json

```diff
@@ -29,7 +29,7 @@
     "requirejs": "2.2.0",
     "requirejs-mustache": "https://github.com/jfparadis/requirejs-mustache.git#5fb8c0a3e8560526e8f9617a689e2924a8532d0a",
     "requirejs-text": "2.0.10",
-    "sinon": "1.17.5",
+    "sinon": "1.17.7",
```

Comment:
side note we should look into updating to sinon 3 at some point. Current version is having trouble with `fakeTimers` sadly.

Reply:
> side note we should look into updating to sinon 3 at some point

Your sinon 3 PR is merged!